packages/create-redwood-app/vitest.config.mts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig, configDefaults } from 'vitest/config'
+
+export default defineConfig({
+  test: {
+    exclude: [...configDefaults.exclude, 'templates/**'],
```

Comment:
Pretty nice that we can just extend from the defaults whereas we had to list all the defaults like `node_modules` before if we were extending from them